### PR TITLE
Learning Paths

### PR DESCRIPTION
**Pull request by flossypurse**
This PR adds a new Learning Path section to the docs which includes a Long-running Human-driven Workflow Learning Path that works in conjunction with this application https://github.com/temporalio/background-checks

**PREVIEW**: https://deploy-preview-794--mystifying-fermi-1bc096.netlify.app/docs/learning-paths/background-checks/project-narrative

### CODE REVIEW

File: docs/learning-paths/long-running-human-driven-workflow/what-are-the-application-requirements.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+We will need a CLI for each user type to represent the experience they would have from a user interface.
+
+[Application CLI reference](/docs/learning-paths/long-running-human-driven-workflow/cli)
+
+### Handling PII
+
+Because we are dealing with PII (Personal Identifiable Information) the application must have a way to encrypt data while it is in transit, and at rest in the system.
+
+### Awaiting
```
Comment (rylandg) [inline docs/learning-paths/long-running-human-driven-workflow/what-are-the-application-requirements.md:92]: Hmm weirdish name for a section

File: docs/learning-paths/long-running-human-driven-workflow/what-are-the-application-requirements.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+The Candidate needs a secure way to provide their SSN and confirm their consent.
+
+The Company would like this to happen all at the same time and it needs to be the first step in the Background Check process. That way, the checks can start immediately and automatically as soon as the Candidate confirms.
+
+However, there is no guarantee of when the Candidate might take this step after the HR person has initiated the Background Check.
+
+Additionally there is no guarantee on how long each check might take.
+
+Therefor the application must be able to maintain the state of the Background Check over many days, and perhaps even weeks, awaiting on data.
```
Comment (rylandg) [inline docs/learning-paths/long-running-human-driven-workflow/what-are-the-application-requirements.md:102]:
```suggestion
Therefore the application must be able to maintain the state of the Background Check over many days, and perhaps even weeks, awaiting on data.
```

**Review by rylandg — COMMENTED** [review]
Love what's happening here.

File: docs/learning-paths/background-checks/project-narrative.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- Even when things went as expected, the percentage of issues and failures at that scale meant that it was not reliable as desired.
+
+Checkr understood that something needed to change to their approach and started looking for solutions.
+In their search, they came across the Temporal Platform.
+
+### Why is the Temporal Platform a great fit for this use case?
+
+Code based workflows appeal to developers.
+Business processes (Background Checks) map directly to Workflow functions.
+This results in a very close mapping of business requirements to code and vice versa.
```
Comment (flossypurse) [inline docs/learning-paths/background-checks/project-narrative.md:63]:
```suggestion
Code based workflows appeal to developers as business processes (Background Checks) map directly to Workflow functions.
This results in a very close mapping of business requirements to code.
```

File: docs/learning-paths/background-checks/cli-reference.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+Example:
+
+```bash
+bgc-company start --email <candidate_email> --package <package_type>
+```
+
+Output:
+
+If successful the output will show "Created check".
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:81]:
```suggestion
If successful, the output shows "Created check".
```

File: docs/learning-paths/background-checks/cli-reference.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+Cancels a Background Check.
+
+Example:
+
+```bash
+bgc-company cancel --email <email> --id <check_id>
+```
+
+Output:
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:106]: Output to come? If no output, delete this heading and an adjacent blank line.

File: docs/learning-paths/background-checks/cli-reference.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+**Example:**
+
+```bash
+bgc-candidate accept --token <token> --ssn <ssn> --address <address> --dob <dob>
+```
+
+**Output:**
+
+If successful the output will show "Accepted".
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:132]:
```suggestion
If successful, the output shows "Accepted".
```

File: docs/learning-paths/background-checks/cli-reference.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+#### `--token` (required)
+
+Background Check Token.
+
+Get this from the email.
+
+#### `--ssn` (required)
+
+Social Security Number of the Candidate.
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:142]:
```suggestion
Social Security number of the Candidate.
```

File: docs/learning-paths/background-checks/cli-reference.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+**Example:**
+
+```bash
+bgc-candidate decline --token <token>
+```
+
+**Output:**
+
+If succesful the output will show "Declined".
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:164]:
```suggestion
If successful, the output shows "Declined".
```

File: docs/learning-paths/background-checks/cli-reference.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+**Example:**
+
+```bash
+bgc-researcher employmentverify --token <token>
+```
+
+**Output:**
+
+If successful the output will show "Employment verification received"
```
Comment (djmagee) [inline docs/learning-paths/background-checks/cli-reference.md:182]:
```suggestion
If successful, the output shows "Employment verification received"
```

File: docs/learning-paths/background-checks/how-to-use.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+This starts up a new Docker Application with a stack of containers.
+
+## How to monitor the application
+
+Once the applications has started you can navigate to Grafana at http://localhost:3000/.
+
+Click the Dashboards->Home button and click on the Temporal dashboard.
+This dashboards contains information about your Temporal server, as well as information about your workflows, activities, persistence, and service client information.
```
[the remark this answers is not in the document]
Comment (smittyweygant) [reply docs/learning-paths/background-checks/how-to-use.md:33]: @flossypurse We're using port 8085 for Grafana in the docker-compose. Recommend we update line 30 to align - Grafana at http://localhost:8085/ 

Regarding your question about Monitoring Metrics and the SDK - metrics are emitted by Temporal Worker services using the Temporal SDK.

File: docs/learning-paths/background-checks/project-narrative.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
+---
+id: project-narrative
+title: Welcome to the background-checks project
+description: TODO
+sidebar_label: Introduction
+---
+
+## What is the goal of this project?
+
+The goal of this project is to teach you, the developer, how to think about building Temporal Applications that have Long Running Human Driven Workflows using a Temporal SDK, by leading you through a comprehensive implementation within the context of a real-life use case.
```
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs/learning-paths/background-checks/project-narrative.md:10]: Not too late to change that.
But then wouldn't it be "Long-running Human-driven Workflow"?
Where the  first letter of the second word in the hyphenation is lowercase?

[the remark this answers is not in the document]
Comment (djmagee) [reply docs/learning-paths/background-checks/project-narrative.md:10]: Welllll...you can get lots of debate on that issue. Both approaches are valid, but _Chicago_ 17th recommends capping the second word too. (I'm also inclined to add a comma following "-running", but that's a separate discussion.)

More specifically, _Chicago_ sez:

1. Always capitalize the first element. 
2. Capitalize any subsequent elements unless they are articles, prepositions, coordinating conjunctions (_and_, _but_, _for_, _or_, _nor_) , or such modifiers as _flat_ or _sharp_ following musical key symbols. 
3. If the first element is merely a prefix or combining form that could not stand by itself as a word (_anti_, _pre_, etc.), do not capitalize the second element unless it is a proper noun or proper adjective.

[the remark this answers is not in the document]
Comment (djmagee) [reply docs/learning-paths/background-checks/project-narrative.md:10]: Having said all that, we're claiming a specific, Temporal-centric meaning for the term, so we can define it the way we want.

**Comment by yoDon** [comment]
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/temporal-0anhvci/reviewers)

File: docs/learning-paths/background-checks/project-narrative.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (flossypurse) [reply docs/learning-paths/background-checks/project-narrative.md:10]: Sounds like we have decided on:
> Human-Driven Long-Running Workflow

**Review by djmagee — APPROVED** [review]

**Pull request merged**